.github/actions/run-e2e/action.yaml

```diff
@@ -72,11 +76,21 @@ runs:
       shell: bash
       run: go install gotest.tools/gotestsum@latest
     - name: Create cluster
+      id: create_cluster
       shell: bash
       run: ref/.github/scripts/create-cluster.sh
       env:
         FLC_NAMESPACE: ${{ inputs.flc-namespace }}
         FLC_ENVIRONMENT: ${{ inputs.flc-environment }}
+    - name: Check cluster status
+      id: check_cluster_status
+      shell: bash
+      run: |
+        echo "Cluster status: ${{ steps.create_cluster.outputs.cluster_status }}"
+        if [[ "${{ steps.create_cluster.outputs.cluster_status }}" == "skipped" ]]; then
```

Comment:
Why are we not checking for failure as well? Or is this done implicitly and I am missing it?